macaroonbakery/bakery.py

```diff
@@ -42,6 +51,10 @@ def discharge_all(macaroon, visit_page=utils.visit_page_with_browser,
     discharge macaroons.
     '''
     discharges = [macaroon]
+    if visit_page is None:
+        visit_page = utils.visit_page_with_browser
+    if jar is None:
+        jar = requests.cookies.RequestsCookieJar()
```

Comment:
What's the benefit of this over having them as the default args in the definition as before?